bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/niri/Niri.java

```diff
@@ -797,20 +793,6 @@ public ROIDescription(ROIDescription roid) {
         }
 
         /**
-         * Return the x start pixel.
-         */
-        public int getXStart() {
-            return _xStart;
-        }
-
-        /**
-         * Return the y start pixel.
-         */
-        public int getYStart() {
-            return _yStart;
-        }
-
-        /**
```

Comment:
In this particular case, if we aren't using the (x,y) start we might as well remove the fields altogether? Also, the class seems to want to be immutable but isn't.  Seems like we could nudge it there without much work.